### ISSUE TITLE
WCAG 2.2:  Consistent Navigation (3.2.3) - Can you break it when you magnify a page?

### ISSUE THREAD

**Issue by Taballe** [w3c/wcag#4711]
Hi,

Let's say that you have a web application that has navigation links along the left side of the page; this is on ALL pages.  **Some** of the pages happen to have a horizontal tabstrip as well.

When I magnify the page:

1. On pages with big tabstrips, the ones that no longer fit are replaced by a "More" tab.  If you open it, it shows the tabs that no longer fit, as a menu.  If there were 10 tabs, the last 3 are moved into the More menu.  This is what I expect to happen.

2. The same thing is done for the vertical navigation links.  HOWEVER, the overall order **does** change. 

- Let's say that there are a total of 10 navigation links.  At 100% magnification, all ten are visible.  At 125% magnification, the More menu button appears, but 10 is not put into this menu.  
- For example, the navigation links will show 1, 2, 3, 4, 5, 6, 10, and then the More menu.  When you open the More menu, you see 7, 8, and 9.

In my opinion, this is a failure of "Consistent Navigation".  The order should not change simply because the user magnified the page.   The more you zoom, the larger the More menu gets, but 10 never makes it into the More menu.

Looking forward to some feedback.  If you agree that this is a valid failure, could this be mentioned in the WCAG 2.2 documentation, for future reference? 

Thanks!
